### DISCUSSION TITLE
我太晚發現的一篇matplotlib.pyplot網誌講解文章

### DISCUSSION THREAD

**Discussion by qeeeee39** [1101-GS3073-C/Python-Community-GS3073-C#72] [General]
資料視覺化在機器學習裡是重要的一環，阿python裡常用來畫圖的模組就是matplotlib啦~

想來分享一篇我至今查到最完整簡單並且解說清楚的的一篇統整matplotlib.pyplot基礎的文章(來自小狐狸事務所的網誌，他也有有更進階的文章)。
http://yhhuang1966.blogspot.com/2020/04/python-matplotlib.html

每次我在畫實驗數據圖都很痛苦，飽受網路資料雜散之苦，用了很多不對的方法走過來，最近看到這篇深深覺得自己學程式的方法不對，我應該也要用「好好研究一個主題」的心態去試每種模組或是語法的撰寫，而不是「拿到就用」。
學習方法的不正確讓我寫了一年的程式直到最近從頭學過才覺得自己寫的很心安理得...... :(

**Comment by ccathyy** [comment]
哇嗚!感謝你分享這篇文章，讓我發現「小狐狸事務所」是一個可以學習到許多新知識的地方^^

**Comment by jim11261314** [comment]
這個對實驗課的幫助莫大，好完整
學期初看的文章，往往是英文
現在有中文可以複習，非常感謝

**Reply by qeeeee39** [reply]
實驗課加油😥
